shuttle/db/db_functions.py

```diff
@@ -334,28 +322,62 @@ def get_on_call_shuttle_logs():
     return results
 
 
+def get_break_logs(date):
+    date = datetime.datetime.strptime(date, '%b-%d-%Y').strftime('%d-%b-%Y')
+    sql = "SELECT * FROM SHUTTLE_BREAK_LOGS"
+    results = query(sql, 'read')
+    logs_for_date = {}
+    logs_for_date_iter = 0
+    # Break logs don't have a log date so we need to split the clock_out time to date
+    # Then we sort it in another method
+    for i in range(len(results)):
+        date_of_log = results[i]['clock_out'].strftime('%d-%b-%Y')
+        if date_of_log == date:
+            logs_for_date[logs_for_date_iter] = results[i]
+            real_name = username_search(logs_for_date[logs_for_date_iter]['username'])
+            logs_for_date[logs_for_date_iter]['name'] = real_name[0]['firstName'] + ' ' + real_name[0]['lastName']
+            logs_for_date_iter += 1
+    return logs_for_date
```

Comment:
What is the purpose of logs_for_date_iter here?

Reply:
So that logs_for_date has all of the logs in the correct order. If I just used i, it would put the value of i in for logs_for_date's iterator which isn't in order from 0. I guess maybe appending it may be a better solution?